.semaphore/semaphore.yml

```diff
@@ -28,13 +34,15 @@ blocks:
         commands:
           - sem-version go 1.15
           # Go project boiler plate
-          - export "SEMAPHORE_GIT_DIR=~/src/${SEMAPHORE_PROJECT_NAME}"
+          - export "SEMAPHORE_GIT_DIR=/tmp/${SEMAPHORE_PROJECT_NAME}"
           - export "PATH=$(go env GOPATH)/bin:${PATH}"
           - mkdir -vp "${SEMAPHORE_GIT_DIR}" "$(go env GOPATH)/bin"
           # Dep install db
           - checkout
           - cache restore deps-$SEMAPHORE_GIT_BRANCH-$(checksum go.sum),deps-$SEMAPHORE_GIT_BRANCH,deps-master
           - go mod download
+          # Run beanstalkd for test
+          - docker run -d -p 11300:11300 schickling/beanstalkd

```

Comment:
Separate it ?

Reply:
Cannot separate it, otherwise the pipeline will fail
ref: https://shopline.semaphoreci.com/workflows/7b10f804-3dbb-4b6d-b0c4-fbbbddde768b?pipeline_id=b16fd045-edf1-4506-b24c-dcf92873ff6a